api/v1alpha1/imageclusterinstall_types.go

```diff
@@ -41,6 +41,12 @@ const (
 
 	InstallSucceededReason  = "ClusterInstallationSucceeded"
 	InstallSucceededMessage = "Cluster installation has succeeded"
+
+	HostValidatedCondition     = "Validated"
+	HostValidationFailedReason = "HostValidationFailed"
+	HostValidationSucceeded    = "HostValidationSucceeded"
```

Comment:
Don't you need pending?

Reply:
i assume we can leave it as unknown? Do we want actually to set pending?